x-pack/plugins/siem/server/lib/hosts/elasticsearch_adapter.ts

```diff
@@ -104,20 +104,32 @@ export const formatHostEdgesData = (
     }
   );
 
-const formatHostItem = (fields: ReadonlyArray<string>, bucket: HostAggEsItem): HostItem =>
-  fields.reduce<HostItem>((flattenedFields, fieldName) => {
+const formatHostItem = (fields: ReadonlyArray<string>, bucket: HostAggEsItem): HostItem => {
+  const a = fields.reduce<HostItem>((flattenedFields, fieldName) => {
     const fieldValue = getHostFieldValue(fieldName, bucket);
     if (fieldValue != null) {
       return set(fieldName, fieldValue, flattenedFields);
     }
     return flattenedFields;
   }, {});
+  return a;
+};
 
 const getHostFieldValue = (fieldName: string, bucket: HostAggEsItem): string | string[] | null => {
   const aggField = hostFieldsMap[fieldName]
     ? hostFieldsMap[fieldName].replace(/\./g, '_')
     : fieldName.replace(/\./g, '_');
-  if (['host.ip', 'host.mac'].includes(fieldName) && has(aggField, bucket)) {
+  if (
+    [
+      'host.ip',
+      'host.mac',
+      'cloud.instance.id',
+      'cloud.machine.type',
+      'cloud.provider',
+      'cloud.region',
+    ].includes(fieldName) &&
```

Comment:
I just randomly deleted this and everything ran fine. I don't know the overall benefit of this as we know what we are sending down and this looks like extra touch points IMO. Up to you and the previous authors though.

Reply:
While the `HostOverview` continues to function, looks like removing those hamstrings the `HostTable` as it also uses this function to get the host field values. Going to leave this one as is.

![image](https://user-images.githubusercontent.com/2946766/57411972-67206280-71ac-11e9-83d9-a21e22977e6b.png)